docs/sections/user_guide/yaml/components/shave.rst

```diff
@@ -39,6 +39,10 @@ Describes the required parameters to run a ``shave`` configuration.
 
   The j/y dimensions of the compute domain (not including halo)
 
+  **output_grid_file:**
+
+  The path to the output file.
```

Comment:
The `shave` program requires that the user name the output file and path, so adding that here.